test_user_views.py

```diff
@@ -0,0 +1,137 @@
+
+from app import app, CURR_USER_KEY
```

Comment:
Wow, all the tests, amazing! 😍 Please put a section in the README describing how to run them.

Reply:
Thanks!!! 👍 
I will update README